### ISSUE TITLE
4.3.1 error: editevent: it's not anymore possible to add event image in frontend

### ISSUE THREAD

**Issue by jojo12** [jemproject/JEM-Project#1864]
@mckillo :

When it's a serie (repeated events): (or with root button or with single event button)

it's only possible in backend to add or remove an image

Now my question what else can't be uploaded in frontend?

EDIT: I tested with root event edit and single event edit

editvenue is no problem!

**Comment by mckillo** [comment]
Let me see it.

**Comment by mckillo** [comment]
@jojo12 Check it.

**Comment by jojo12** [comment]
works now in frontend and backend!

**Comment by jojo12** [comment]
yes, it works,
but I don't understand all effects in series and single!

**Comment by mckillo** [comment]
Yes, that is so, JEM doesn't have control of inheritance between parents and children, previously not allowed either, except that the children couldn't be changed. If the parent was changed, all children were deleted and rewritten with the saved content with new ids.

Now, this part makes them independent, but a control for inheritance is still missing (TODO --> If the parent is changed, the content of any remaining children should be updated accordingly).

**Comment by mckillo** [comment]
A checkbox needs to be implemented to either follow or break the inheritance regarding the root event and add to control code to management it.
